angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html

```diff
@@ -5,6 +5,7 @@
       tooltip-trigger="mouseenter"
       tooltip-placement="top"
       tooltip-popup-delay="500"
+      onClick="_gaq.push(['_trackEvent', 'Widgets', 'List Mode', 'List Mode']);"
```

Comment:
I think it would be better to just add a call to [miscService.pushGAEvent](https://github.com/UW-Madison-DoIT/angularjs-portal/blob/master/angularjs-portal-frame/src/main/webapp/portal/misc/services.js#L29) in the `switchMode` function.